src/config-loader.js

```diff
@@ -256,11 +264,12 @@ class ConfigLoader {
    * Find files by pattern
    * @param {String} pattern
    * @param {String} path
+   * @param {String[]} ignore
    * @returns {*}
    * @private
    */
-  _find(pattern, path) {
-    return glob.sync(pattern, { cwd: path, absolute: true, dot: true, ignore: ConfigLoader.IGNORE_PATTERNS });
+  _find(pattern, path, ignore = []) {
+    return glob.sync(pattern, { cwd: path, absolute: true, dot: true, ignore: ConfigLoader.IGNORE_PATTERNS.concat(ignore) });
```

Comment:
change `IGNORE_PATTERNS` `get` method to
```
return this.getProjectConfig().ignore || [];
```
and remove `ingore` from method args